lang/python/python3-sqlparse/Makefile

```diff
@@ -0,0 +1,38 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sqlparse
+PKG_VERSION:=0.3.0
+PKG_RELEASE:=1
+
+PYPI_NAME:=sqlparse
+PKG_HASH:=7c3dca29c022744e95b547e867cee89f4fce4373f3549ccd8797d8eb52cdb873
+
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=Peter Stadler <peter.stadler@student.uibk.ac.at>
```

Comment:
I suggest moving above PKG_LICENSE.

Reply:
Changed that (I used the above [example](https://github.com/openwrt/packages/pull/10242#issuecomment-541939101)).

Reply:
What I meant was like this: https://github.com/openwrt/packages/blob/master/libs/libevhtp/Makefile#L24

Reply:
Sorry, yes I did something wrong in pushing the other commit. Now it should be right. Thank you.